### DISCUSSION TITLE
How to use "HasTotal" extension in OrchardCore Application

### DISCUSSION THREAD

**Discussion by aminmohammadi05** [EasyOC/EasyOC#25] [Q&A]
Hello
I am using OrchardCode 1.6 and desperately need to have total count of items in searches and graphql queries. I searched for EasyOC packages in nuget manager and found bunch of them including "EasyOC.Extensions" but I do not know how to use them in my application.
Besides when I clone EasyOC repository I could not compile it as to it refers to version 1.5-preview. Would you please upgrade it to version 1.6?
Thank you in advance

**Comment by hyzx86** [comment]
@aminmohammadi05  You can refer to this line to update the local code, and if everything works, you can also commit your changes to this repo, thanks

https://github.com/EasyOC/EasyOC/blob/8f08dd48105981f4b0233c8b5ef2ee1fae9981e2/src/EasyOC.Build/Dependencies.props#L6C19-L6C19

**Reply by aminmohammadi05** [reply]
Thank you very much.
I have a problem in returning n random items by a Lucene query. I do not know how does it possible. I have tried this dsl query:
```
{
  "query": {
    "term": { "Content.ContentItem.ContentType": "BlogPost" }
  },
  "sort": [
    { "Content.ContentItem.CreatedUtc": { "order": "random" } }
  ],
  "size": 10
}
```
but that is no use.

**Reply by hyzx86** [reply]
Maybe the OC doesn't support the "order" : "random" syntax,

but you can try `Elasticsearch Module`  https://docs.orchardcore.net/en/latest/docs/reference/modules/Elasticsearch/
